scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1907,6 +1907,30 @@ class FormatOps(
     }
   }
 
+  // Optional braces in templates after `:|with`
+  object OptionalBracesTmpl {
+    def unapply(meta: FormatToken.Meta): Option[Template] =
+      meta.leftOwner match {
+        case t: Template if templateCurly(t).contains(tokens(meta.idx).left) =>
+          Some(t)
+        case _ => None
+      }
+  }
+
+  // Optional braces after any token that can start indentation:
+  // )  =  =>  ?=>  <-  catch  do  else  finally  for
+  // if  match  return  then  throw  try  while  yield
+  object OptionalBracesBlck {
```

Comment:
```suggestion
  object OptionalBracesBlock {
```
 it's just a letter more?